src/config.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  connectionWrite,
+  createConnection,
+  createConnectionTls,
+} from './connection';
+import { createDB } from './skytable';
+import { bufferToHandshakeResult, getClientHandshake } from './protocol';
+import type {
+  ConnectionOptions as ConnectionTLSOptions,
+  TLSSocket,
+} from 'node:tls';
+import { Socket } from 'node:net';
+
+/**
+ * Configuration for a client connection (single node)
+ */
+export class Config {
+  private username: string;
+  private password: string;
+  private host: string;
+  private port: number;
+  private connection: Socket | TLSSocket | undefined;
```

Comment:
Please `export default` the `Config` class here

Reply:
We can make the changes in index.ts.